spec/forms/password_reset_email_form_spec.rb

```diff
@@ -12,29 +12,33 @@
         user = build(:user, :signed_up, email: 'test1@test.com')
         subject = PasswordResetEmailForm.new('Test1@test.com')
 
-        result = {
-          success: true,
-          errors: [],
+        extra = {
           user_id: user.uuid,
           role: user.role,
           confirmed: true,
         }
 
+        result = instance_double(FormResponse)
+
+        expect(FormResponse).to receive(:new).
```

Comment:
In this form unit test, we want to test the output of the `submit` method, which in this case, is an instance of the `FormResponse` class. The form doesn't know that the response is used for analytics (by calling `.to_h` on the object), nor should it care. That's a controller concern.

We do want to make sure that the `FormResponse` class is called because any other result would be invalid. We require and expect the `submit` method to return a `FormResponse` object. That's the only object that is expected. 

We don't care about the implementation of `FormResponse` in this test because it is tested separately. However, we do care that this class is being instantiated with the proper arguments.